package.json

```diff
@@ -6,11 +6,16 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
+    "dotenv": "^16.0.3",
```

Comment:
if we gonna use the file ```APIs``` instead of ```.env``` so you can go ahead and remove the package ```dotenv``` from the ```package.json```